packages/meson.build

```diff
@@ -1,9 +1,11 @@
 conf_pkg = configuration_data()
 
 if host_machine.system() == 'windows'
-    installer_setup = find_program(meson.project_source_root() / 'tools/win-installer-setup.ps1', )
+    installer_setup = find_program(meson.project_source_root() / 'tools/win-installer-setup.ps1')
     run_target('win-installer',
                command: [installer_setup, meson.project_build_root(), meson.project_source_root()])
+    portable_setup = find_program(meson.project_source_root() / 'packages/win_installer/portable/create-portable.ps1')
```

Comment:
This reminds me that I should really PR meson to use ExecutionPolicy Bypass by default when running powershell programs...

Reply:
Opened https://github.com/mesonbuild/meson/issues/8790, I'll PR a fix some other time.